Source/Platform/FileTests.cpp

```diff
@@ -166,10 +166,12 @@ TEST( PlatformFileTest, PathIsNotAbsolute )
 	ASSERT_TRUE( IsAbsolute( fullPath.c_str() ) );
 }
 
+#if HELIUM_ASSERT_ENABLED 
 TEST( PlatformFileTest, FailToMakePath )
 {
-	ASSERT_FALSE(MakePath(""));
+	ASSERT_DEATH(MakePath(""), "");
```

Comment:
where is this macro coming from?  what does it do?  is it a google test macro?

Reply:
Yes, it is part of the GTest.

https://github.com/abseil/googletest/blob/master/googletest/docs/advanced.md#how-to-write-a-death-test